site/docs/components/toast/examples.mdx

```diff
@@ -49,6 +49,13 @@ Use the success toast to communicate the successful completion of an action. A s
 
 </LivePreview>
 
+<LivePreview componentName="toast" exampleName="CustomIcon" >
+
+## Custom Icon
+
+Use the `Icon` prop to customize the icon shown in the toast, while still respecting the status prop.
```

Comment:
This is still saying `Icon`, also status is not back-ticked so is not formatted.
Can we rephrase (use the LLM) , respecting might not be my first choice of grammar?